opendebates/views.py

```diff
@@ -239,7 +239,12 @@ def vote(request, id):
     )
     if created:
         # update the DB with the real tally
-        Submission.objects.filter(id=id).update(votes=F('votes')+1)
+        Submission.objects.filter(id=id).update(
+            votes=F('votes')+1,
+            local_votes=F('local_votes')+(
+                1 if voter.state and voter.state == get_local_votes_state()
+                else 0)
+        )
```

Comment:
Are you okay with the implications that this is only generated at vote time and thus the setting can never be changed during a run?